code/datums/uplink_item.dm

```diff
@@ -22,13 +24,38 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 				uplink_items[I.category] = list()
 
 			uplink_items[I.category] += I
+			if(I.limited_stock < 0 && !I.cant_discount && I.item && I.cost > 1)
+				sales_items += I
 
 		for(var/datum/uplink_item/I in last)
 			if(!uplink_items[I.category])
 				uplink_items[I.category] = list()
 
 			uplink_items[I.category] += I
 
+	for(var/i in 1 to 3)
+		var/datum/uplink_item/I = pick_n_take(sales_items)
+		var/datum/uplink_item/A = new I.type
+		var/discount = 0.5
+		A.limited_stock = 1
+		I.refundable = FALSE
+		A.refundable = FALSE
+		if(A.cost >= 20)
+			discount *= 0.5 // If the item costs 20TC or more, it's only 25% off.
+		A.cost = max(round(A.cost * (1-discount)),1)
+		A.category = "Discounted Gear"
+		A.name += " ([round(((initial(A.cost)-A.cost)/initial(A.cost))*100)]% off!)"
+		A.job = null // If you get a job specific item selected, actually lets you buy it in the discount section
+		A.reference = "DIS[newreference]"
+		A.desc += " Only [A.limited_stock] in stock. Normally costs [initial(A.cost)] TC. All sales final"
```

Comment:
You forgot a period at the end.

Reply:
Just delete the 'all sales final' from here, and change the 'Discounted Gear' header to 'Discounted Gear (no refunds!)'. That way you're not having to repeat the no refunds idea 3 times, once in each item.